stacker/exceptions.py

```diff
@@ -102,6 +111,34 @@ def __init__(self, stack_name, output, *args, **kwargs):
         super(OutputDoesNotExist, self).__init__(message, *args, **kwargs)
 
 
+class DestroyWithoutNotificationQueue(Exception):
+
+    def __init__(self, stack_name, *args, **kwargs):
+        self.stack_name = stack_name
+        message = "You are attempting to destroy `%s` " % stack_name
```

Comment:
btw, rather than doing the += method, you can also represent this with paranthesis,and it tends to look a bit cleaner:

```
message = (
    "You are attempting to "
    "destroy `%s`"
)
```

etc